src/index.ts

```diff
@@ -218,7 +218,11 @@ prog
       // Install deps
       process.chdir(projectPath);
       const safeName = safePackageName(pkg);
-      const pkgJson = generatePackageJson({ name: safeName, author });
+      const pkgJson = generatePackageJson({
+        safeName,
+        author,
+        pkg,
```

Comment:
```suggestion
        name: pkg,
        safeName,
        author,
```

It really should be called `pkgName` or something, but that's a refactor, so we're stuck with `pkg` in `create` for now, but should not use that poor naming elsewhere if avoidable. `pkg` itself is overloaded a few times within this file too.